charts/bulk-scan-processor/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: bulk-scan-processor
 home: https://github.com/hmcts/bulk-scan-processor
-version: 0.1.4
+version: 0.1.5
```

Comment:
Not using latest java chart ?

Reply:
this is chart version :) I will do update Docker file and java chart version as a separate PR(might include Java 11 as well)